browse-scaler/src/cmr.js

```diff
@@ -5,6 +5,14 @@ const cmrRootUrl = `http://${process.env.CMR_ROOT}`;
 const cmrCollectionUrl = `${cmrRootUrl}/search/collections.json?concept_id=`;
 const cmrGranuleUrl = `${cmrRootUrl}/search/granules.json?concept_id=`;
 
+/**
+ * fetchConceptFromCMR: Given a concept id, fetch the metadata supplied by
+ * the elasticsearch JSON response
+ * @param {String} conceptId A collection or granule concept-id
+ * @param {String} cmrEndpoint The collection or granule search URL. This is set
+ * as a variable so you don't have to think about what it is
```

Comment:
This is confusing. I'm not sure what "set as a variable so you don't have to think about it" means.

Reply:
Yeah you're right - removed